lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -817,29 +818,20 @@ export class UserAgentApplication {
    * @ignore
    */
   public isInIframe() {
-      return window.parent !== window;
-  }
-
-  /**
-   * @hidden
-   * Returns whether parent window exists and has msal
-   */
-  private parentIsMsal() {
-    return window.parent !== window && window.parent.msal;
-  }
+    return window.parent !== window;
```

Comment:
nit: consider using `window.frameElement` for additional security.
https://developer.mozilla.org/en-US/docs/Web/API/Window/frameElement

It also returns null if the parent and the embedded element is not the same origin, supporting the case where we want to load an iFrame inside of an iFrame.

Reply:
Ah didn't know about that, I'll take a look at it.

Reply:
Although I think we would want to know about the parent if we are in a cross domain scenario?